src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,162 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
-    @Override
-    public void add(T value) {
+    private static final int ZERO_SIZE = 0;
```

Comment:
Private methods and classes should be after public ones in your class. Consider moving the `Node` class to the end of the `MyLinkedList` class.